### PR TITLE
#1996 Update to graphql ^16.2.0

### LINKED ISSUE #1996: Update to graphql ^16.2.0
It's about time to update graphql to the newest possible.

```
  '16.2.0': '2021-12-17T14:57:40.667Z'
```

It's would be good to check all queries if they're do alright and compare outputs before & after.

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Do a quick check before the merge.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #1996
- [x] Tested well on local, works fine, build is successful as well, didn't have to do any specific version related changes 

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases

### Had issue bounty label ?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI, a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by yangwao** [comment]
Did you have checked outputs before and after if things are okay?

What's was breaking changes between previous version we had, what's was impact, if any?

**Comment by prachi00** [comment]
> Did you have checked outputs before and after if things are okay?
> 
> What's was breaking changes between previous version we had, what's was impact, if any?

Yes I did, didn't find anything that would break anything

File: package.json [hunk 1]
```diff
@@ -64,7 +64,10 @@
     "css-loader": "4.3.0",
     "date-fns": "^2.28.0",
     "emoji-unicode": "^2.0.1",
-    "graphql": "^15.8.0",
+    "file-saver": "^2.0.5",
+    "freezeframe": "^5.0.2",
+    "glob-parent": "^6.0.2",
```
Comment (vikiival) [inline package.json:69]:
```suggestion

```

File: nuxt.config.js [hunk 2]
```diff
@@ -239,7 +239,10 @@ export default {
   apollo: {
     clientConfigs: {
       ...defineApolloConfig(),
-      subsquid: toApolloEndpoint(process.env.SUBSQUID_ENDPOINT || 'https://app.gc.subsquid.io/beta/rubick/004/graphql'),
+      subsquid: toApolloEndpoint(
+        process.env.SUBSQUID_ENDPOINT ||
+          'https://app.gc.subsquid.io/beta/rubick/004/graphql'
+      ),
```
Comment (vikiival) [inline nuxt.config.js:245]: formatting

**Review by vikiival — CHANGES_REQUESTED** [review]
May merge the latest main?

File: nuxt.config.js (hunk 2 above)
Comment (prachi00) [reply nuxt.config.js:245]: @vikiival this happened automatically by the linter on commit

**Review by vikiival — APPROVED** [review]

**Comment by yangwao** [comment]
Made future-proof issue to tackle this while upgrades will come tho https://github.com/kodadot/nft-gallery/issues/2028

Perfect, I’ve sent the payout https://kusama.subscan.io/extrinsic/0xa0287f6628fe2a589f2de662c48a177c34c64b5181800167e7221c575dc83e19

Let’s grab another issue and get rewarded! 
https://github.com/kodadot/nft-gallery/issues/

**Pull request merged**